draft-ietf-rats-corim.md

```diff
@@ -1429,9 +1429,176 @@ is extended using Endorsements etc. from the accepted tags.
 
 ### Matching Evidence against Reference Values
 
+An Endorser may use CoMID tags to publish Conditional Endorsements, which
+are added to the Accepted Claims Set only if specified conditions apply.
+This section describes the process performed by the Verifier to determine
+which Conditional Endorsements from the candidate CoMIDs should be added
+to the Accepted Claims Set.
+
+The verifier checks whether Conditional Endorsements are applicable by
+comparing Evidence in the Accepted Claims Set against Reference Values
+from the CoMID. These Reference Values may be provided as Reference Value
+Triples or may be combined with the Endorsements, for example as the
+Conditional Endorsement Series Triple.
+
+The following subsections describe how the CoRIM tells the verifier which
+Reference Values and Endorsed Values are grouped together ({{sec-grouping-ref-vals}})
+and how the verifier matches a Reference Value against the Accepted Claims Set
+({{sec-match-all-ref-vals}}).
+
+#### Grouping Reference Values and Endorsements {#sec-grouping-ref-vals}
+
+> This paragraph will be replaced by a description of how the CoRIM tells the
+verifier which Reference Values and Endorsements are grouped together.
+
+[^issue]: Need to get agreement on how group membership is encoded. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/136
+
+[^issue]: Need to describe how to match conditional endorsements. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/80
+
+#### Matching all Reference Values in a group against the Accepted Claims Set {#sec-match-all-ref-vals}
+
+If all Reference Values in a group match entries in the Accepted Claims Set
+then all Endorsements in the group are added to the Accepted Claims Set
+(see {{sec-add-to-acs}}). {{sec-match-one-ref-val}} describes how one
+Reference Value is matched against the Accepted Claims Set.
+
+If any Reference Value in a group does not match the Accepted Claims Set then
+all Endorsements in the group are silently ignored.
+
+Each group is processed independently of other groups. If a group fails to match
+the Accepted Claims Set then this does not affect the processing of other groups.
+
+#### Matching a Reference Value against the Accepted Claims Set {#sec-match-one-ref-val}
+
+This section describes how a Reference Value is matched against Evidence in the Accepted
+Claims Set.
+If any part of the processing indicates that the Reference Value does not match then the remaining steps in this section are skipped for that group.
+
+A Reference Value consists of an `environment-map` plus a `measurement-map`. In the
+`reference-values-triple-record` these are packaged together. In other triples multiple
+Reference Values are represented more compactly by letting one `environment-map`
```

Comment:
```suggestion
Reference Values pertaining to an environment are represented more compactly by collecting various types of measurements in a single `measurement-map`
```

Reply:
I think there are overlapping multi-line suggested changes. Can we close on this PR and manage changes using more focused issues? i.e., open an issue rather than suggest changes here.

Reply:
Tracked via issue #144